src/Pages/Profile/MemberView/ChangePassword.js

```diff
@@ -52,13 +58,26 @@ export default function ChangePasswordModal(props) {
             id="confirm"
             value={confirmPassword}
             name="confirm"
-            type="email"
+            type={showPassword ? 'text' : 'password'}
             onChange={(e) => {
               setConfirmPassword(e.target.value);
             }}
             className={INPUT_CLASS_NAME}
           />
         </div>
+
+        <div className="form-control mt-1">
+          <label className="label cursor-pointer">
+            <span className="label-text">Show Password</span>
+            <input
+              type="checkbox"
+              className="toggle"
+              checked={showPassword}
+              onChange={togglePassword}
```

Comment:
then you dont need the helper function
```suggestion
              onChange={() => setShowPassword(!showPassword)}
```

Reply:
done